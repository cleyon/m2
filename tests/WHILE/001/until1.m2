@sequence X create
@sequence X set 10
@until @X@ < 5
Iteration @X--@
@endwhile
Out of while loop
