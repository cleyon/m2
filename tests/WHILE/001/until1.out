Iteration 10
Iteration 9
Iteration 8
Iteration 7
Iteration 6
Iteration 5
Out of while loop
